src/test/java/JP2ImageConverter/services/KakaduServiceTest.java

```diff
@@ -391,6 +415,74 @@ public void testKduCompressRotatedJpg() throws Exception {
         assertContains("2832x4256", output);
     }
 
+    @Test
+    public void testKduCompressYCbCrTif() throws Exception {
+        String mockedTif = String.valueOf(new File( tmpFolder + "/mockedImage.tif"));
```

Comment:
there's an extra space in there, but I'd also suggest using path functionality rather than converting to a File and using String.valueOf. So `tmpFolder.resolve("mockedImage.tif").toString()`